convert.go

```diff
@@ -93,11 +94,14 @@ func FromIP(ip net.IP) (ma.Multiaddr, error) {
 	return FromIPAndZone(ip, "")
 }
 
-// DialArgs is a convenience function returning arguments for use in net.Dial
+// DialArgs is a convenience function that returns network and address as
+// expected by net.Dial. See https://godoc.org/net#Dial for an overview of
```

Comment:
The set of possible return values laid out in [net#Dial](https://godoc.org/net#Dial) is more than we currently implement in this function. Would be good to mention that we don't support `unix`, `unixgram`, and `unixpacket`?

Reply:
or just `unix*` for brevity.